e2e-tests/functions

```diff
@@ -5,14 +5,33 @@ ROOT_REPO=${ROOT_REPO:-$(realpath ../../..)}
 source "${ROOT_REPO}/e2e-tests/vars.sh"
 test_name=$(basename "$(pwd)")
 
+if oc get projects >/dev/null; then
+	# Guessing Openshift version from master node API version
+	case "$(oc get nodes --no-headers=true | grep master | head -n1 | grep -Eo 'v[0-9]+\.[0-9]+')" in
+		"v1.11") OPENSHIFT=3.11 ;;
+		*) OPENSHIFT=4 ;;
+	esac
+fi
+
 create_namespace() {
 	local namespace=$1
-	if [[ ${CLEAN_NAMESPACE} == 1 ]]; then
+	if [[ -n $OPENSHIFT ]]; then
+		if [[ -n $OPERATOR_NS && $(oc get project "$OPERATOR_NS" -o json | jq -r '.metadata.name') ]]; then
```

Comment:
`-n` is not needed here just like above, also, I think the second part of the comparrison must be outside of the square brackets, like this:
`if [[ "$OPERATOR_NS" ]] && oc get project "$OPERATOR_S" -o json > /dev/null 2>&1 | jq -r '.metadata.name' > /dev/null 2>&1; then`
because the command `oc get projec ...` would return some string which will always be `true`